caddytls/storagetest/storagetest_test.go

```diff
@@ -0,0 +1,39 @@
+package storagetest
```

Comment:
Possibly a way to mock out the filesystem: https://talks.golang.org/2012/10things.slide#8